backend/src/api/api.ts

```diff
@@ -12,6 +12,7 @@ import {
   coursesToMentors,
```

Comment:
```suggestion
  mentor,
```
create a mentor service please. `coursesToMentors` should be inside